src/index.html

```diff
@@ -5,8 +5,353 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header__link">
+
+        <img
+         src="./images/logo.png"
+         alt="Moyo"
+         class="header__image"
+        >
+      </a>
+
+      <nav class="navigation">
+        <ul class="navigation__list">
+          <li class="navigation__element">
+            <a href="#" class="navigation__link is-active">Apple</a>
+          </li>
+          <li class="navigation__element">
+            <a href="#" class="navigation__link">Samsung</a>
+          </li>
+          <li class="navigation__element">
+            <a href="#" class="navigation__link">Smartphones</a>
+          </li>
+          <li class="navigation__element">
+            <a
+              href="#"
+              class="navigation__link"
+              data-qa="nav-hover">Laptops & Computers
+            </a>
```

Comment:
don't forget about code formatting
```suggestion
              data-qa="nav-hover"
            >
              Laptops & Computers
            </a>
```